test/E2ETests/E2EApps/E2EApp/E2EApp.csproj

```diff
@@ -34,8 +42,10 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.WebUtilities" Version="2.2.0" />
-    <PackageReference Condition="$(TestBuild) == 'true'" Include="Microsoft.Azure.Functions.Worker" Version="1.0.0" />
-    <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.3.0" />
-    <PackageReference Include="System.Text.Json" Version="6.0.5" />
+    <PackageReference Condition="$(TestBuild) == 'true'" Include="Microsoft.Azure.Functions.Worker" Version="1.10.0" />
+    <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.7.0" />
+  
+    <PackageReference Condition="$(FunctionsRuntimeVersion) != '3'" Include="System.Text.Json" Version="6.0.5" />
```

Comment:
I did not understand the reasoning behind handling only 2 cases here - version = 3 and != 3 and not handling any other versions? Do we expect any change in this file once V5 is released, for example?

Reply:
Reasoning is that if nothing is provided (!=3) we will default to 4 - and yes we have changes once v5 is released, and not sure how long we will continue testing v3, etc. 

This design was discussed in a previous design meeting as the likely fastest way to continue testing v3 and v4, but in the future we may want to test with a multi-targeted app or other designs.

Reply:
Sounds good. Thanks.